vaadin-text-field.html

```diff
@@ -139,11 +142,14 @@
 
       [part="label"] {
         display: block;
+        position: absolute;
```

Comment:
This change causes the label to not reserve space anymore either, and causes fields to not have any white-space between them when stacked.

Before:
<img width="211" alt="screen shot 2018-05-24 at 9 36 40" src="https://user-images.githubusercontent.com/66382/40468423-0fddc492-5f36-11e8-9051-918d9ddbbb82.png">

After:
<img width="209" alt="screen shot 2018-05-24 at 9 37 02" src="https://user-images.githubusercontent.com/66382/40468428-15472d06-5f36-11e8-8866-28e185f58b8e.png">

Reply:
Fixed. Without absolute I couldn’t make a consistent alignment in both flex and line baseline contexts. So restored the spacing with padding and margin on the host.

I’m using symmetrical spacing, padding on the top and margin on the bottom, so that:
- they don’t collapse if stacked in a block context, 
- they look centered inside bars with narrow height (e. g., app search bar, vaadin-grid-sorter, etc)